src/java.xml/share/classes/com/sun/org/apache/xml/internal/serializer/dom3/LSSerializerImpl.java

```diff
@@ -606,6 +631,16 @@ public void setParameter(String name, Object value) throws DOMException {
                 } else {
                     fDOMConfigProperties.setProperty(DOMConstants.S_XSL_OUTPUT_OMIT_XML_DECL, "yes");
                 }
+            } else if (name.equalsIgnoreCase(Constants.ORACLE_JAXP_PROPERTY_PREFIX
+                    + DOMConstants.S_IS_STANDALONE)) {
+                fFeatures = state ? fFeatures | IS_STANDALONE : fFeatures & ~IS_STANDALONE;
+                if (state) {
+                    fDOMConfigProperties.setProperty(DOMConstants.S_JDK_PROPERTIES_NS
+                            + DOMConstants.S_IS_STANDALONE, "yes");
```

Comment:
Multiple concatenations of the same strings, seems awkward at best and perhaps a maintenance burden.
Would it make sense to declare them as static strings?
Though in this case, it could be a single call to setProperty with the value being `(state ? "yes" : "no")`

Reply:
Will do. I'll see if we should do the same with the existing ones.